src/amo/components/RatingManager/styles.scss

```diff
@@ -32,8 +32,29 @@
     justify-content: space-between;
   }
 
-  .AddonReviewCard-ratingOnly .AddonReviewCard-allControls {
-    justify-content: center;
+  .AddonReviewCard-ratingOnly {
+    .AddonReviewCard-allControls {
+      justify-content: center;
+    }
+
+    .AddonReviewCard-delete {
+      text-align: center;
+      width: 100%;
+
+      .ConfirmButton-buttons {
+        display: block;
+      }
+
+      .ConfirmButton-cancel-button,
+      .ConfirmButton-confirm-button {
```

Comment:
Can you also make the font sizes match across all of these buttons?

<img width="309" alt="screenshot 2018-10-01 16 39 36" src="https://user-images.githubusercontent.com/55398/46317466-af353200-c598-11e8-8a29-376fc713b56c.png">

Since there are a lot of other puffy buttons, the easiest thing is probably to match the big font size. Here's an idea:

```diff
diff --git a/src/amo/components/RatingManager/styles.scss b/src/amo/components/RatingManager/styles.scss
index dfa5e2de2..4d989ec0a 100644
--- a/src/amo/components/RatingManager/styles.scss
+++ b/src/amo/components/RatingManager/styles.scss
@@ -47,6 +47,7 @@
 
       .ConfirmButton-cancel-button,
       .ConfirmButton-confirm-button {
+        font-size: $font-size-puffy-buttons;
         margin: 0;
         width: 100%;
       }
diff --git a/src/core/css/inc/vars.scss b/src/core/css/inc/vars.scss
index 4b7ecbb80..c03c91132 100644
--- a/src/core/css/inc/vars.scss
+++ b/src/core/css/inc/vars.scss
@@ -26,6 +26,7 @@ $font-size-l: 28px;
 $font-size-xl: 36px;
 $font-size-xxl: 42px;
 $font-size-form-header: 24px;
+$font-size-puffy-buttons: 15px;
 
 // Animation Variables
 $transition-short: 150ms;
diff --git a/src/ui/components/Button/styles.scss b/src/ui/components/Button/styles.scss
index 60bc2ca85..044d13126 100644
--- a/src/ui/components/Button/styles.scss
+++ b/src/ui/components/Button/styles.scss
@@ -2,7 +2,6 @@
 
 $micro-font-size: 11px;
 $default-font-size: 13px;
-$puffy-font-size: 15px;
 
 /* Button */
 @mixin button(
@@ -47,7 +46,7 @@ $puffy-font-size: 15px;
 
     &.Button--puffy {
       border-radius: 4px;
-      font-size: $puffy-font-size;
+      font-size: $font-size-puffy-buttons;
       height: auto;
       min-height: 48px;
       padding: 0 16px;
@@ -55,8 +54,8 @@ $puffy-font-size: 15px;
       .Icon {
         @include margin-end(8px);
 
-        height: $puffy-font-size;
-        width: $puffy-font-size;
+        height: $font-size-puffy-buttons;
+        width: $font-size-puffy-buttons;
       }
     }
 
```

Reply:
That works well, thanks!